webhook.py

```diff
@@ -221,16 +252,19 @@ def change_update_mode(sender_psid, mode):
             id,
             msbot.constants.ASAP_MODE_CMD
         ),
+        msbot.constants.INFO_CMD: lambda id: info(id),
     }
 
     message = received_message.lower()
     if message in responses:
         resp = responses[message](sender_psid)
     else:
-        resp = msbot.constants.RESP_INVALID_UNSUBBED
+        resp = RESP_INVALID_CMD
         if database.user_exists(sender_psid):
-            resp = msbot.constants.RESP_INVALID_SUBBED
-        resp = to_text_response(resp)
+            resp = text_quick_reply_response(
+                msbot.constants.RESP_INVALID_SUBBED,
+                [ INFO_BUTTON ]
+            )
 
     send_message(sender_psid, resp)
```

Comment:
man pythons so weird to me. you can use `resp` even though it was declared in the if/else block?

Reply:
yeah I know right

Reply:
https://stackoverflow.com/questions/2829528/whats-the-scope-of-a-variable-initialized-in-an-if-statement